resources/CropService.php

```diff
@@ -98,7 +104,11 @@ private function getAttachment($attachmentId)
      */
     private function setFocusPoint($focusPoint)
     {
-        $this->focusPoint = $focusPoint;
+        $this->focusPoint = [50, 50];
```

Comment:
Please write down this variable `[50, 50]` as a private variable outside the function and in the top of the class. Should be shared with JS later on as well